queries.sql

```diff
@@ -23,4 +23,25 @@ SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name NOT LIKE 'Gabumon';
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
-SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+-- second milestone
+
+-- How many animals are there?
+SELECT COUNT(*) from animals;
+
+-- How many animals have never tried to escape?
+SELECT COUNT(*) from animals WHERE escape_attempts = 0;
+
+-- What is the average weight of animals?
+SELECT AVG(weight_kg) from animals;
+
+-- Who escapes the most, neutered or not neutered animals?
+SELECT SUM(escape_attempts), neutered from animals GROUP BY neutered;
```

Comment:
- [ ] Good job so far 👏. I think for this query, it would be better if you use the `AVG` operation or  `MAX` in order to find out which ones escaped the most. 
HINT: It might look like this 
```sql
  SELECT neutered, AVG(escape_attempts) FROM animals
   GROUP BY neutered;
```

Reply:
Suggested changes applied :heavy_check_mark: 
![Screenshot from 2022-09-13 11-03-52](https://user-images.githubusercontent.com/37988027/189937272-e00e3e29-8737-4772-bb69-457e9b2abcf7.png)

Reply:
- [ ] Just as the previous reviewer mentioned here, please make sure to use the `AVG` operation instead of the `SUM`. So far, I don't see any change related to that issue in this file, so please make sure to work on that.